cli/pom.xml

```diff
@@ -49,5 +49,9 @@
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why do `cli` depend on `coordinator`?

Reply:
I fixed this issue. this is because I need to use the application object, this object I put it in common.